.circleci/config.yml

```
@@ -230,9 +232,9 @@ jobs:
           command: |
             python -m venv env
             . env/bin/activate
-            pip install -r requirements.txt
+            pip install << parameters.dependency-versions >> --upgrade --only-binary << parameters.only-binary >>
```

Does it not matter if using `=` or not?
```suggestion
            pip install << parameters.dependency-versions >> --upgrade --only-binary=<< parameters.only-binary >>
```

---

It's just the name, doesn't matter what characters we put in it.